coding-concepts.js

```diff
@@ -37,23 +39,23 @@ const languages = ["JavaScript", "Ruby", "Python", "C++"]
 const index = 1
 // console.log(languages[index])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: "Ruby"
+// b) Verify and explain:Indexes start from 0 starting "JavaScript"
 
 
 // --------------------4) What will this log?
 
 const weekendDays = ["saturday", "sunday"]
 // console.log(weekendDays.toUpperCase())
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: Changes the letters to uppercase. 
+// b) Verify and explain: nothing happened. It needs to be put in as a string first out of the array. 
 
 
 // --------------------5) What will this log?
 
 const dataTypes = ["number", "string", "Boolean"]
 // console.log(typeof dataTypes.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: wrong answer due to the name miss spelled. 
+// b) Verify and explain: with the name corrected, we get "3", the total string length in the array.
```

Comment:
Did you check line 58 in the terminal?

dataTypes.length will be 3 because there are 3 values within the array. Values within an array are separated by the comma. 

typeof 3 will be number because the datatype of 3 is number